src/wasm-type.h

```diff
@@ -165,12 +165,12 @@ class Type {
   static Type get(unsigned byteSize, bool float_);
 
   // Returns true if left is a subtype of right. Subtype includes itself.
-  static bool isSubType(Type left, Type right);
+  static bool isSubType(const Type& left, const Type& right);
```

Comment:
I have a half-baked (or, more precisely, quarter-baked) patch for bringing reference types proposal up to date in my local machine, which I started some weeks ago and didn't get around to finish. I'm now trying to pick up the patch.. Do you need that changes for GC? I don't want to block you from proceeding to wait for my patch, but at the same time, I think it is good to add that reference type patch separately from GC and make sure all existing (and to-be-updated) tests the fuzzer pass with reference types proposal, because currently the fuzzer is testing reference type instructions already.

I was away for most of the last week. I was gonna make this change this week, but please let me know if your plan is blocked by this or interfere with this schedule, in which case we can change the plan. I wasn't able to take a look at this patch yet, but I think I'll soon.

Reply:
The GC proposal [restores](https://github.com/WebAssembly/gc/blob/master/proposals/gc/Overview.md#type-grammar) `anyref` and `nullref`, so all we really need to do right now is add an `externref` type. But some current uses of `anyref` might need to use `externref` instead. @aheejin, you'll have a better idea than me whether it's better to rename the current `anyref` to `externref` then add a new `anyref` or to just add a new `externref` to what we already have.

Reply:
I'm confused.
- Does the GC proposal really restores `nullref`? After the recent reference type changes, every reference type can be nullified, or in other words, have a null value. Why would we need to restore a separate `nullref` type? I think what you quoted is just not up-to-date, no?
- I think we should add `anyref`; `anyref` and `externref` seem to be two separate types. `anyref` is a supertype of all reftypes while `externref` is not.

Reply:
Iiuc there is [no `nullref` anymore](https://github.com/WebAssembly/gc/blob/master/proposals/gc/MVP.md#reference-types-1).

Reply:
Aha, I've been looking at the wrong GC document. Thanks for the clarification, @dcodeIO!